datavalid.yml

```diff
@@ -447,12 +451,14 @@ schemas:
       - name: uof_uid
         description: >
           unique identifier of use of force report.
-      - name: use_of_force_type
+      - name: use_of_force_description
+      - name: use_of_force_reason
       - name: use_of_force_level
       - name: use_of_force_effective
       - name: age
       - name: years_of_service
       - name: officer_injured
+      - name: citizen_age
```

Comment:
Let's make the use of force tables highly normalized. The web team will find the same columns repeated across tables confusing. I think `use_of_force_description` and `use_of_force_reason` only need to stay in the `use_of_force` table. `citizen_age` only need to stay in `use_of_force_citizens` table.

Reply:
Sorry, I didn't tag you correctly in this issue from the web team where he asked for the tables to be repeated. Please see #292